pullrequest.js

```diff
@@ -1,17 +1,60 @@
 
 var isGitHub = $("meta[property='og:site_name']").attr('content') === 'GitHub';
 
+function htmlIsInjected() {
+  return $('.collapse-lines').length !== 0;
```

Comment:
I think we should use some hidden or empty element that we insert at the top of the body with the id 'pretty-pull-requests', slightly more defensive in case github ever introduces this collapse-lines class. We could even then use document.getElementById and it should be as fast as possible to check if the html is injected already.